main.py

```diff
@@ -7,7 +7,7 @@
 import logging
 import smtplib
 import kcidb
-
+from kcidb.cache import Client
```

Comment:
Let's follow the pattern of just reaching to the client from `kcidb` package, as everywhere else her. This saves us a lot of imports as this file uses basically the complete functionality of KCIDB, because this is where it all comes together. That is, do not import `kcidb.cache` directly, and refer to the client via `kcidb.cache.Client`.

Reply:
Ah!!! got it